src/ui/components/common/nav.rs

```diff
@@ -142,19 +169,29 @@ pub fn TopNav() -> impl IntoView {
 #[component]
 pub fn BottomNav() -> impl IntoView {
   let i18n = use_i18n();
+
+  let QueryResult { data, refetch, .. } = use_site_state();
+
+  let instance_api_version =
+    Signal::derive(move || data().map_or_else(|| None, |res| Some(res.ok()?.version)));
```

Comment:
This might be simpler as:
```suggestion
    Signal::derive(move || data().map(|res| res.ok().map(|res| res.version)));
```